linker/Mono.Linker.Steps/AddBypassNGenStep.cs

```diff
@@ -0,0 +1,133 @@
+﻿//
+// AddBypassNGenStep.cs
+//
+// Author:
+//   Jb Evain (jbevain@gmail.com)
+//
+// (C) 2006 Jb Evain
+// (C) 2007 Novell, Inc.
+//
```

Comment:
Is this correct header for new files?

Reply:
You should just use a header with your name and Microsoft copyright

Reply:
We don't normally put the author's name or Microsoft copyright on open source files under .NET foundation. For example: https://github.com/dotnet/coreclr/blob/master/src/jit/morph.cpp#L1 or
https://github.com/dotnet/corefx/blob/master/src/System.Collections/src/System/Collections/Generic/LinkedList.cs#L1 . Should we use a similar pattern for linker sources?

Reply:
@russellhadley Any opinion on this?

Reply:
.NET foundation works too

Reply:
Updated the header.